.jscsrc

```diff
@@ -1,5 +1,6 @@
 {
 	"preset": "jquery",
 	"maximumLineLength": null,
-	"requireCamelCaseOrUpperCaseIdentifiers": null
+	"requireCamelCaseOrUpperCaseIdentifiers": null,
+	"requireCapitalizedComments": false
```

Comment:
And if you know, why the lines I mentioned are commented out in the first place ?